recipes-devtools/ghc/ghc-native_8.10.7.bb

```diff
@@ -18,8 +18,12 @@ SRC_URI += " \
 # the installation from the host rootfs.
 #
 
+BUILD_RANLIB_remove = "-D"
+
 do_configure() {
-    ./configure --prefix=${prefix} --enable-shared
+    ./configure --prefix=${prefix} \
+                CC=`which gcc` \
+                LD=`which ld`
```

Comment:
These seem wrong - I wish there was more detail on why CC & LD can't be found.

Reply:
OE builds up the `CC` var with `CC=${BUILD_CC}` which translates to
```
${CCACHE}${BUILD_PREFIX}gcc ${BUILD_CC_ARCH}
```
this results in "gcc " with a space.  The actual commands that invoke the binary `gcc` are fine with the space, but it looks like this bit with `deriveConstants` passing in "gcc " as the argument to --gcc-program
```
inplace/bin/deriveConstants --gen-haskell-type -o includes/dist-derivedconstants/header/GHCConstantsHaskellType.hs --tmpdir includes/dist-derivedconstants/header/ --gcc-program "gcc " --gcc-flag -Wall
...
| deriveConstants: gcc : rawSystem: posix_spawnp: does not exist (No such file or directory)
```
Removing the space was enough to get it past this part without failing, so I think Niko just interpreted this as "can't find gcc" when it was a bit more nuanced than that.

Reply:
Ok, sounds good.  Thanks for the details.